python/LoadExternalSTAC/LoadLandsatSTAC.ipynb

```diff
@@ -0,0 +1,890 @@
+       "       5679565.0, 5679555.0, 5679545.0, 5679535.0,\n",
+       "       ...\n",
+       "       5675165.0, 5675155.0, 5675145.0, 5675135.0, 5675125.0, 5675115.0,\n",
+       "       5675105.0, 5675095.0, 5675085.0, 5675075.0],\n",
+       "      dtype=&#x27;float64&#x27;, name=&#x27;y&#x27;, length=456))</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-437b901c-c102-413f-80cf-22e7f19869ce' class='xr-section-summary-in' type='checkbox'  checked><label for='section-437b901c-c102-413f-80cf-22e7f19869ce' class='xr-section-summary' >Attributes: <span>(4)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'><dt><span>Conventions :</span></dt><dd>CF-1.9</dd><dt><span>institution :</span></dt><dd>openEO platform - Geotrellis backend: 0.40.1a1</dd><dt><span>description :</span></dt><dd></dd><dt><span>title :</span></dt><dd></dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset> Size: 2GB\n",
+       "Dimensions:        (t: 1141, x: 363, y: 456)\n",
+       "Coordinates:\n",
+       "  * t              (t) datetime64[ns] 9kB 2014-01-04 2014-01-13 ... 2023-12-28\n",
+       "  * x              (x) float64 3kB 6.396e+05 6.396e+05 ... 6.432e+05 6.432e+05\n",
+       "  * y              (y) float64 4kB 5.68e+06 5.68e+06 ... 5.675e+06 5.675e+06\n",
+       "Data variables:\n",
+       "    crs            |S1 1B b''\n",
+       "    var            (t, y, x) float32 755MB nan nan nan ... 0.004835 0.00841\n",
+       "    unkown_band_1  (t, y, x) float32 755MB 0.05641 0.05791 0.05791 ... nan nan\n",
+       "Attributes:\n",
+       "    Conventions:  CF-1.9\n",
+       "    institution:  openEO platform - Geotrellis backend: 0.40.1a1\n",
+       "    description:  \n",
+       "    title:        "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import xarray as xr\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "ndvi = xr.load_dataset(\"Merged_NDVI.nc\")\n",
+    "ndvi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2c5fb9be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we downloaded them as separate bands, so now lets merge them for a single plot\n",
+    "\n",
+    "\n",
+    "def nanmedian_two_bands(b1, b2):\n",
+    "    return np.nanmedian(np.array([b1, b2]), axis=0)\n",
+    "\n",
+    "\n",
+    "# Use apply_ufunc to apply np.nanmedian element-wise\n",
+    "nanmedian_band = xr.apply_ufunc(\n",
```

Comment:
is it possible to perform this type of processing with openeo processes instead?